schema/logs/README.md

```diff
@@ -0,0 +1,223 @@
+  - allowed  - indication that this event was subsequently allowed by some authority system
+  - change   - indication that this event is related to something that has changed
+  - connection - indication that this event is related to network traffic with indication of connection activity
+  - creation   - indication that this event is related to resources being created
+  - deletion   - indication that this event is related to resources being deleted
+  - denied     - indication that this event is related to resources being denied access
+  - error      - indication of an error related event
+  - group      - indication of events that are related to group objects
+  - info       - indication of events which are informative without other distinct classification 
+  - installation   - indication that this event is related to resources being installed 
+  - protocol   - indicate that the event is related to specific knowledge of protocol info
+  - end        - indicate that te event is related to some termination state
+  - start      - indicate that te event is related to some initiation state
+  - user       - indicate that the event is related to specific knowledge a user resource
+
+
+`event.result` gives a success or a failure indication from the perspective of the entity that produced the event. (ECS driven)
+
+**Possible values**
+
+ - failure
+ - success
+ - pending
+ - undetermined
+
+
+#### Exception Fields
+This field encapsulated the exception that should appear under the event section `event.exception` (OTEL driven)
+
+ - `message`: The exception message.
+ - `stacktrace`: A stacktrace as a string in the natural representation for the language runtime. The representation is to be determined and documented by each language SIG.
+ - `type`: The type of the exception (its fully-qualified class name, if applicable). The dynamic type of the exception should be preferred over the static type in languages that support it.
+
+---
+
+### Overview
+
+In observability, Logs are typically unstructured data that is generated by applications or systems as a record of events or messages.
+Logs can contain any information that the emitting application or system wants to include, and they often contain free-form text rather than structured data.
+This makes logs difficult to process and analyze automatically, but it also provides a lot of flexibility and versatility in terms of what information can be captured.
+
+According to [ECS](https://github.com/elastic/ecs) and the most recent (experimental) [OTEL](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions) definitions, we formalized a unified
+log schema. This schema can be used for working with a well-structured set of typed logs arriving from categorical sources.
+These sources are expected to report information in a specific way that will simplify future correlations and consolidate similar concerns.
+
+### Classifications
+
+#### HTTP 
```

Comment:
We had some similar definitions added for synthetics project: https://github.com/opensearch-project/dashboards-observability/issues/75

Reply:
nice - this Classifications convention follows ECS protocol

Reply:
@YANG-DB are we also, planning to support cookies, certificates info, etc in schema for http requests?